src/cli/test_setup.rs

```diff
@@ -0,0 +1,79 @@
+use crate::{
+    cli::{keygen, KeygenArgs},
+    config::{NetworkConfig, PeerConfig},
+};
+use clap::Args;
+use std::{
+    error::Error,
+    fs::{self, DirBuilder},
+    iter::zip,
+    path::PathBuf,
+};
+
+#[derive(Debug, Args)]
+#[clap(
+    name = "test_setup",
+    about = "Prepare a test network of three helpers",
+    next_help_heading = "Test Setup Options"
+)]
+pub struct TestSetupArgs {
+    #[arg(short, long, default_value = "test_data")]
+    output_dir: PathBuf,
+
+    #[arg(long)]
+    https: bool,
```

Comment:
I might prefer the inverse, so that doing the good thing is easier.  So maybe:

```suggestion
    disable_https: bool,
```

Reply:
Thoughts on whether it makes sense to do this on `helper` itself too? (I think I considered doing that, but resisted the temptation to tweak something that already existed even though I agree HTTPS should be the default.)

Aside: do you happen to know how to configure clap for `--foo` / `--no-foo` boolean options from a single specification?